packages/manager/apps/veeam-backup/mocks/iam.mock.ts

```diff
@@ -1,15 +1,21 @@
-import { PathParams } from 'msw';
-import { Handler } from '../../../../../playwright-helpers';
-import resourceList from './iam.json';
+import { IamCheckResponse } from '@ovh-ux/manager-react-components';
+import { backupList } from './veeam-backup.mock';
 
-const findResourceByUrn = (params: PathParams) =>
-  resourceList.find(({ resourceURN }) => resourceURN === params.urn);
-
-export const getIamMocks = (): Handler[] => [
+export const iamResponse: IamCheckResponse[] = [
+  {
+    urn: backupList[0].iam.urn,
+    authorizedActions: [
+      'account:apiovh:iam/resource/edit',
+      'account:apiovh:service/terminate',
```

Comment:
you can use your `iamActions` constant on  `packages/manager/apps/veeam-backup/src/veeam-backup.config.ts`

Reply:
Yes bien vu